clients/feeder/feeder.go

```diff
@@ -18,6 +18,17 @@ import (
 	"github.com/NethermindEth/juno/utils"
 )
 
+type ErrorCode string
```

Comment:
I think we can just return plain `error`s and not worry about the json structure. See other comment.